src/main/java/net/mcreator/ui/workspace/selector/WorkspaceSelector.java

```diff
@@ -338,7 +339,25 @@ private void reloadRecents() {
 						removeRecentWorkspace(defaultListModel.elementAt(idx));
 						reloadRecents();
 					} else if (mouseEvent.getClickCount() == 2) {
-						workspaceOpenListener.workspaceOpened(recentsList.getSelectedValue().getPath());
+						int n = JOptionPane.YES_OPTION;

```

Comment:
This will only show the dialog when the workspace is open through the workspace selector, which is one of the many ways where the workspace can be open and thus this message will not be shown in all cases